datetime/mod.ts

```diff
@@ -6,6 +6,18 @@
  *
  * This module is browser compatible.
  *
+ * ```
```

Comment:
Maybe better to add the language tag?

```suggestion
 * ```ts
```